.github/workflows/benchmark_asterinas.yml

```diff
@@ -51,6 +51,7 @@ jobs:
           # File-related benchmarks
           - lmbench/ramfs_create_delete_files_0k_ops
           - lmbench/ramfs_create_delete_files_10k_ops
+          - lmbench/ramfs_copy_files_bw
```

Comment:
Do you want to display the result in the summary table? If so, you need to edit `summary.json`.